src/ScottPlot4/ScottPlot/Renderable/Legend.cs

```diff
@@ -244,4 +355,10 @@ public void UpdateLegendItems(Plot plot, bool includeHidden = false)
             }
         }
     }
+
+    public enum LegendOrientation
```

Comment:
I forgot to mention in the main body of my review, but we already have an `Orientation` enum here: https://github.com/ScottPlot/ScottPlot/blob/81735684448d35d0939712e1f5389d4e4e2c3e35/src/ScottPlot4/ScottPlot/Renderable/Orientation.cs

Reply:
I found two Orientation enums, the second is here: https://github.com/ScottPlot/ScottPlot/blob/81735684448d35d0939712e1f5389d4e4e2c3e35/src/ScottPlot4/ScottPlot/Enums/Orientation.cs

I deleted the one in Renderable/Orientation and kept the one in Enums/Orientation

Reply:
I don't think you can delete the one in Renderable/Orientation, that would be a breaking change.

It looks like that enum is unused now, and the last reference I can find to it comes from ScottPlot 4.0, the last release to which was in August 2021, having been made obsolete by 4.1 in May 2021. Even so, since it's public, it could be used by consumers of the library, and I don't want to break their code.